doc/changes/changes_3.0.1.md

```diff
@@ -1,11 +1,19 @@
-# Test Database Builder 3.0.1, released 2021-MM-DD
+# Test Database Builder 3.0.1, released 2021-02-24
+
+Code name: Made `script.execute` robust against quotes
 
 ## Summary
 
+TDB allows you to create and execute Exasol scripts. In this released a bug in the implementation of execute that lead to an exception when string parameters containing quotes were passed to the execute function.
```

Comment:
```suggestion
TDB 3.0.1 allows you to create and execute Exasol scripts. In this released a bug in the implementation of execute that lead to an exception when string parameters containing quotes were passed to the execute function.
```
It's easier to copy this into the community announcement when the version number is in that description.